spec/inputs/gelf_spec.rb

```diff
@@ -1,8 +1,22 @@
 require "logstash/devutils/rspec/spec_helper"
 require "logstash/inputs/gelf"
+require_relative "../support/helpers"
 require "gelf"
 
-describe "inputs/gelf" do
+Thread.abort_on_exception = true
+describe LogStash::Inputs::Gelf do
+  context "when interrupting the plugin" do
+    let(:port) { rand(1024..65535) }
```

Comment:
Okay, this is what I propose.

- I'll change rand for Flores::Random
- Fix flores to use the rspec seed, which is okay in that case since its an explicitl rspec helper I assume it to have repeatable run.

If people want pure random with no way to repeat it, they can fallback to `Kernel.rand` and manage the seed themself. What do you think? I think this work isn't part of this PR.